dataset/reentrancy/reentrancy_cross_function.sol

```diff
@@ -14,7 +14,7 @@ contract Reentrancy_cross_function {
     function transfer(address to, uint amount) {
         if (userBalances[msg.sender] >= amount) {
             userBalances[to] += amount;
-        u   serBalances[msg.sender] -= amount;
+            userBalances[msg.sender] -= amount;
```

Comment:
strange change. why did this happen?

Reply:
I would suggest to have this fixed as another PR -- as it is not related to this one.

Reply:
Ok, how can I delete just this file from this PR?

Reply:
you need to revert the changes in the file. (git revert, maybe).